modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+Components of {product-title-short} use various default ports and connection protocols. Depending on your system and firewall configuration, you might need to configure your firewall to allow traffic on certain ports.
+
+The following table provides default ports and protocols for some connections within {product-title-short} and between {product-title-short} and external components. This is helpful for configuring your firewall to allow traffic to and from the cluster. However, you might need more detailed information in some scenarios. For example, if your firewall is integrated in the cluster router, you might need to specify ports for connections that happen within one cluster but might be on different IP networks. In this scenario, you can use the {product-title-short} network policy YAML files in your {ocp} and Kubernetes clusters to determine connections and ports that you might need to configure.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and Sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by Sensor to Central's port 443.
+
+| Central and user (CLI)
+a| * gRPC
+* HTTPS (with `--force-http1` option)
+| 443
+| For more information on the `--force-http1` option, see the `roxctl` command options in "roxctl CLI command reference".
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+| Connects to definitions.stackrox.io` by default.
+
+| Scanner to Sensor
```

Comment:
This connection happens within the secured cluster - not relevant for the firewall.